<launch>
    <arg name="map_frame" default="map" />
    <arg name="base_link_frame" default="base_footprint" />

    <node name="map_to_baselink" pkg="map_to_baselink" type="map_to_baselink" output="screen">
        <param name="map_frame" type="string" value="$(arg map_frame)" />
        <param name="base_link_frame" type="string" value="$(arg base_link_frame)" />
    </node>
</launch>